clients/java/client/src/test/java/org/operaton/bpm/client/impl/TopicSubscriptionManagerTest.java

```diff
@@ -39,9 +39,7 @@
 
 import static org.awaitility.Awaitility.await;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
`org.junit.jupiter.api` is JUnit **5** API, not JUnit 4, which the referenced issue was about.

However, these changes are also fine. AssertJ is used widely now in the codebase, and refactorings towards AssertJ are fine.